static/css/devhub/base.less

```diff
@@ -229,10 +289,23 @@ section {
       }
     }
   }
+}
 
+.DevHub-BlogPosts {
   .read-more {
     text-decoration: none;
-    color: @tertiary-font-color;
+    color: @color-scheme-dark-items;
     font-weight: 600;
   }
 }
+
+.DevHub-Tools {
+  ul {
+    li {
+      a {
+        font-size: 24px;
```

Comment:
Epic nesting. Do you need this many selectors? Maybe a direct class would be better here to avoid building up the specificity? Also as an aside if there's nothing else requiring nesting here a direct selector would be clearer.